wicket-util/pom.xml

```diff
@@ -40,9 +40,14 @@
 			<groupId>org.apache.commons</groupId>
 			<artifactId>commons-collections4</artifactId>
 		</dependency>
+		<!-- Commented out due to all classes are copied into wicket-commons-fileupload
 		<dependency>
 			<groupId>org.apache.commons</groupId>
 			<artifactId>commons-fileupload2</artifactId>
+		</dependency-->
+		<dependency>
+			<groupId>org.apache.wicket</groupId>
+			<artifactId>wicket-commons-fileupload</artifactId>
```

Comment:
Since we have the dependency here I think there is no need to have it in wicket-core too. It will come as a transitive one